jobmaster/dm/task_manager_test.go

```diff
@@ -23,6 +23,8 @@ const (
 func TestMain(m *testing.M) {
```

Comment:
I forget this function in previous PR. Does it act like SetUpSuit? Is this all your purpose?

Reply:
change to suit